src/components/FeatureViews/Scheduler/EditSessionView.js

```diff
@@ -40,6 +40,7 @@ function EditSessionView({ course, session, editSelection }) {
         category: "",
         is_confirmed: "",
         duration: "",
+        tempDuration: ""
```

Comment:
isn't this just the end time?

Reply:
No, this is because when a user chooses an option, it will create the end_time. But I thought of a case where a user made a mistake and needed to change the duration. That would add to the current end time because of how the switch statement is created. So I created a tempDuration to hold the string until a user submits the form and the final temp duration will become the end_time. If you have another more eloquent way of coding this lmk.

Reply:
I see, that makes sense. The code is fine, it's the naming. Call it "updatedDuration" then. "tempDuration" doesn't mean anything.